src/main/interface/msp.c

```diff
@@ -2535,7 +2549,7 @@ static mspResult_e mspCommonProcessInCommand(uint8_t cmdMSP, sbuf_t *src, mspPos
         {
             uint8_t font_data[64];
             const uint8_t addr = sbufReadU8(src);
-            for (int i = 0; i < 54; i++) {
+            for (int i = 0; i < 64; i++) {
```

Comment:
This will write garbage into NVM with old configurator version. It is probably best to use all data supplied and pad to NVM_RAM_SIZE with zeroes